L1Trigger/TrackFindingTracklet/interface/IMATH_TrackletCalculatorDisk.h

```diff
@@ -0,0 +1,389 @@
+#ifndef L1Trigger_TrackFindingTracklet_interface_IMATH_TrackletCalculatorDisk_h
+#define L1Trigger_TrackFindingTracklet_interface_IMATH_TrackletCalculatorDisk_h
+
+#include "Settings.h"
+#include "imath.h"
+
+//
+// Constants used:
+//   dphisector
+//   rmaxL6
+//   zmaxD5
+//   rmaxdisk
+//   kr, kphi1, kz
+//
+//   rmean[], zmean[]
+
+class IMATH_TrackletCalculatorDisk {
+public:
+  IMATH_TrackletCalculatorDisk(const Trklet::Settings* settings, imathGlobals* globals, int i1, int i2)
+      : settings_(settings), globals_(globals) {
+#ifndef CMSSW_GIT_HASH
+    edm::LogVerbatim("Tracklet") << "=============================================";
+    char s[1024];
+    sprintf(s, "IMATH Tracklet Calculator for Disk %i %i dphisector = %f", i1, i2, settings->dphisector());
```

Comment:
See above for the use of `boost::format` or `snprintf` instead of `sprintf`.

Reply:
now using snprintf(...)